cli/cmd/volume.go

```diff
@@ -0,0 +1,168 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+const (
+	helpVolumeCmd       = "Gluster Volume Management"
```

Comment:
Placed it separately only to facilitate easy translation of these messages in future.